org.elysium.ui/src/org/elysium/ui/preferences/LilyPondValidatorConfigBlock.java

```diff
@@ -0,0 +1,105 @@
+/*******************************************************************************
+ * Copyright (c) 2015 AKSW Xturtle Project, itemis AG (http://www.itemis.eu).
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ ******************************************************************************/
+//code adapted from de.itemis.tooling.xturtle.ui.preferences.TurtleValidatorConfigBlock of the AKSX-Xturtle project
+package org.elysium.ui.preferences;
+
+import java.util.Arrays;
+
+import javax.inject.Inject;
+
+import org.eclipse.core.resources.IProject;
+import org.eclipse.core.resources.ResourcesPlugin;
+import org.eclipse.core.runtime.jobs.Job;
+import org.eclipse.jface.preference.IPreferenceStore;
+import org.eclipse.swt.widgets.Composite;
+import org.eclipse.xtext.ui.editor.preferences.IPreferenceStoreAccess;
+import org.eclipse.xtext.ui.preferences.OptionsConfigurationBlock;
+import org.eclipse.xtext.ui.validation.AbstractValidatorConfigurationBlock;
+import org.elysium.validation.IssueCodes;
+
+@SuppressWarnings("restriction")
+public class LilyPondValidatorConfigBlock extends AbstractValidatorConfigurationBlock {
+
+	private static final String[] otherErrors=new String[]{"ignore","info","warning","error"};
+	private static final String[] otherErrorLabels=new String[]{"no Validation","Info","Warning","Error"};
```

Comment:
I would rather write "Ignore" instead of "no Validation" to be consistent with e.g. JDT.